src/python/pants/option/global_options.py

```diff
@@ -65,6 +65,13 @@ def register_bootstrap_options(cls, register):
                       'pants.backend.python',
                       'pants.backend.jvm',
                       'pants.backend.codegen',
```

Comment:
By errors I mean "module not found" kind of error, as "pants.backend.codegen" is not loaded in this case.

But I have tried and it seems no such error is generated.